pkg/ccl/importccl/import_stmt.go

```diff
@@ -99,6 +100,12 @@ const (
 	avroSchema    = "schema"
 	avroSchemaURI = "schema_uri"
 
+	pgDumpIgnoreAllUnsupported     = "ignore_unsupported"
+	pgDumpIgnoreShuntFileDest      = "ignored_stmt_log"
+	pgDumpUnsupportedSchemaStmtLog = "unsupported-schema-stmts"
```

Comment:
Let's change these separators from `-` to `_` for consistency with the other ones.

Reply:
Done.